src/components/ui/html-renderer/index.web.tsx

```
@@ -0,0 +1,160 @@
+    window.addEventListener('message', handleMessage);
+    return () => window.removeEventListener('message', handleMessage);
+  }, [onLinkPress, handleMessage]);
+
+  const flatStyle = StyleSheet.flatten([styles.container, style]) as Record<string, unknown>;
+
+  // When onLinkPress is provided we need allow-scripts so the click-interceptor runs
+  const sandboxValue = onLinkPress ? 'allow-same-origin allow-scripts' : 'allow-same-origin';
+
+  return (
+    <View style={flatStyle} key={rendererKey}>
+      {/* eslint-disable-next-line react-native/no-inline-styles -- iframe not a RN element */}
```

ESLint

[error] Definition for rule 'react-native/no-inline-styles' was not found.

(react-native/no-inline-styles)